curated_transformers/tokenization/bbpe_encoder.py

```diff
@@ -0,0 +1,107 @@
+from typing import List, TypeVar
```

Comment:
```suggestion
from typing import List, Optional
```
Reverting unintended change (`TypeVar` is not used anywhere).

Reply:
I think this came from the commit that shouldn't have been there. Fixed now.